test/GitUtilities.js

```diff
@@ -65,6 +65,23 @@ describe("GitUtilities", () => {
         "git", ["add", "foo"], opts
       );
     });
+    it("works with absolute path for cwd", () => {
+      const cwd = path.resolve("test");
+      const opts = { cwd };
+      GitUtilities.addFile("foo", opts);
+      expect(ChildProcessUtilities.execSync).lastCalledWith(
+        "git", ["add", "foo"], opts
+      );
+    });
+    it("works with absolute paths for file and cwd", () => {
+      const cwd = path.resolve("test");
+      const file = path.resolve("test", "foo");
```

Comment:
Shouldn't this be `const file = path.resolve(cwd, "foo");`? Otherwise I don't see how this test is any different than the previous one.

Reply:
@evocateur 

In the previous test only `cwd` is absolute; the file path is still relative.

In this one both are absolute.

Your line of code is equivalent to mine, i.e. `path.resolve(path.resolve("test"), "foo")` (your code but with `cwd` inlined) returns the same value as `path.resolve("test", "foo")`.

Should I change it?